resources/views/vendor/laravel-form-builder/collection.php

```diff
@@ -36,20 +36,54 @@ class="<?= strtolower($options['label']) === "narrative" ? $options['options']['
                 </div>' : '';
             $label = strtolower(str_replace(' ', '-', $options['label']));
             $error = '';
+            $errorSection = '';
+
+
+
+            if(isset($options['options']['info_text']) && !empty($options['options']['info_text']))
+            {
+                $error = '<div class="text-danger-error">' . $options['options']['info_text'] . '</div>';
+                $errorSection = '<section class="collection_error">' . $error . '</section>';
+            }
+
+
+            if(isset($options['options']['warning_info_text']) && !empty($options['options']['warning_info_text']))
+            {
+                $errorSection = '<div class="mt-2 flex items-center bg-eggshell pt-2 pr-4 pb-2 pl-4 text-xs rounded-md font-normal">
+                                    <svg
+                                        class="elements-svg"
+                                        width="18"
+                                        height="18"
+                                        viewBox="0 0 18 18"
+                                        fill="none"
```

Comment:
add svg to different file then use it as such: <svg-vue icon="svg name" class="mr-2"></svg-vue>